src/posh-winfetch.ps1

```diff
@@ -290,6 +291,21 @@ function info_uptime {
 }
 
 
+# ===== RESOLUTION =====
+function info_resolution {
+    Add-Type -AssemblyName System.Windows.Forms
+    $Displays = New-Object System.Collections.Generic.List[System.Object];
+    foreach ($monitor in [System.Windows.Forms.Screen]::AllScreens) {
+        $Displays.Add("$($monitor.WorkingArea.Size.Width)x$($monitor.WorkingArea.Size.Height)");
+    }
+
+    return @{
+        title   = "Resolution"
+        content = $Displays -join ' '
+    }
+}
+
+
```

Comment:
The resolution is not including the taskbar for me, 2560x1040

Reply:
Thanks, fixed!